common/app/model/ShareLinks.scala

```diff
@@ -167,5 +168,18 @@ final case class ShareLinks(
     }
 
     ShareLinks.create(sharePlatform, href = href, title = contentTitle, mediaPath = None)
+  }
+
+  def elementShares(elementId: String, mediaPath: Option[String]): Seq[ShareLink] = elementShareOrder.map( sharePlatform => {
+    val webUrlParams = campaignParams(sharePlatform)
+    val href = metadata.webUrl.addFragment(elementId).appendQueryParams(webUrlParams + ("page" -> s"with:$elementId"))
+    ShareLinks.create(sharePlatform, href = href, title = metadata.webTitle, mediaPath = mediaPath)
   })
+
+  val pageShares: ShareLinkMeta = {
+    pageShareOrder match {
+      case (visible, hidden) =>
+        ShareLinkMeta(sharesToLinks(visible), sharesToLinks(hidden))
```

Comment:
I would directly pass the platform list here rather than using an intermediary tuple, that is used nowhere else
```
ShareLinkMeta(sharesToLinks(List(Facebook, Twitter, Email)), 
              sharesToLinks(List(LinkedIn, PinterestPage, GooglePlus, WhatsApp, Messenger))
)
```